xbmc/interfaces/legacy/ListItem.h

```diff
@@ -640,6 +640,7 @@ namespace XBMCAddon
       /// | set           | string (Batman Collection) - name of the collection
       /// | setoverview   | string (All Batman movies) - overview of the collection
       /// | tag           | string (cult) or list of strings (["cult", "documentary", "best movies"]) - movie tag
+      /// | version       | string (Movie version, "EXTENDED" for example)
```

Comment:
Before this goes in we need changelog tags for the python api for v19. Will need `@python_v19 Added version infolabel for movies`